packages/jest-cli/src/run_jest.js

```diff
@@ -75,6 +75,12 @@ const processResults = (runResults, options) => {
   return options.onComplete && options.onComplete(runResults);
 };
 
+// eslint-disable-next-line prefer-const
```

Comment:
why is this needed?

Reply:
eslint didn't seem to recognize that this will be changed in NotifyReporter

Reply:
You mutate the content, not the variable itself, can't this be `const`?

Reply:
Oh looks like I just learned something https://jack.ofspades.com/es6-const-not-immutable/index.html

Reply:
Yeah, there's no true native immutability in JS :(